indexer/services/ender/src/scripts/handlers/dydx_stateful_order_handler.sql

```diff
@@ -141,5 +141,44 @@ BEGIN
     ELSE
         RAISE EXCEPTION 'Unkonwn sub-event type %', event_data;
     END IF;
+
+    /* For order replacement, remove old order */
+    IF event_data->'orderReplacement' IS NOT NULL THEN
```

Comment:
this code won't ever execute because of the earlier return statement on line 93.

Reply:
Thank you for catching! I fixed and adding the check for the old order tests that this is now working properly